### PR TITLE
Package ocaml-protoc.1.2.5

### PR DESCRIPTION
**Pull request by mransan**
### `ocaml-protoc.1.2.5`
Protobuf compiler for OCaml
Protobuf compiler for OCaml

---
* Homepage: https://github.com/mransan/ocaml-protoc
* Source repo: git+https://github.com/mransan/ocaml-protoc.git
* Bug tracker: https://github.com/mransan/ocaml-protoc/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by mransan** [comment]
Ok so I was trying to publish a package after 2 years of absence from the OCaml community. The build system is just make targets which are using ocamlbuild. Is this still supported. 

Error message is indicating a missing target but it is there in the Makefile.

**Comment by mransan** [comment]
Well time to learn dune I guess

**Comment by kit-ty-kate** [comment]
The build system is not the issue. The issue here is that the `url` section is missing so opam doesn't know where to look for it and assumes there is no archive (hence the missing makefile target error)

**Comment by kit-ty-kate** [comment]
My guess is that you might have forgot to push your git tag before calling `opam publish` or something like that

File: packages/ocaml-protoc/ocaml-protoc.1.2.5/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  [make "bin.install" "PREFIX=%{prefix}%" "BINDIR=%{bin}%"]
+]
+remove:  [make "uninstall" "PREFIX=%{prefix}%" "BINDIR=%{bin}%"]
+depends: [
+  "ocaml" {>="4.02.1"}
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "ppx_deriving_protobuf"
+]
+available: [ opam-version >= "1.2" ]
```
Comment (kit-ty-kate) [inline packages/ocaml-protoc/ocaml-protoc.1.2.5/opam]:
```suggestion
```
This is not necessary

File: packages/ocaml-protoc/ocaml-protoc.1.2.5/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+build: [
+  [make "lib.byte"]   
+  [make "lib.native"]
+  [make "bin.native"]
+]
+install: [
+  [make "lib.install" ]
+  [make "bin.install" "PREFIX=%{prefix}%" "BINDIR=%{bin}%"]
+]
+remove:  [make "uninstall" "PREFIX=%{prefix}%" "BINDIR=%{bin}%"]
```
Comment (kit-ty-kate) [inline packages/ocaml-protoc/ocaml-protoc.1.2.5/opam]:
```suggestion
```
This is not necessary since opam 2.0

**Comment by mransan** [comment]
@kit-ty-kate  thanks for the suggestion, let me incorporate those!

**Comment by mransan** [comment]
next attempt at https://github.com/ocaml/opam-repository/pull/15160

**Pull request closed without merging**